miso-web/src/main/webapp/scripts/list_container.js

```diff
@@ -123,6 +123,6 @@ ListTarget.container = {
   },
   searchTermSelector: function(searchTerms) {
     return [searchTerms['created'], searchTerms['changed'], searchTerms['creator'], searchTerms['changedby'], searchTerms['platform'],
-        searchTerms['index_name'], searchTerms['index_seq']]
+        searchTerms['index_name'], searchTerms['index_seq'], searchTerms['kitname']]
```

Comment:
Please add a similar line in `list_library.js`

Reply:
I believe it's already there.